Mathlib/Analysis/SpecialFunctions/Trigonometric/Bounds.lean

```diff
@@ -53,6 +49,68 @@ theorem sin_lt {x : ℝ} (h : 0 < x) : sin x < x := by
   norm_num
 #align real.sin_lt Real.sin_lt
 
+lemma sin_le (hx : 0 ≤ x) : sin x ≤ x := by
+  obtain rfl | hx := hx.eq_or_lt
+  · simp
+  · exact (sin_lt hx).le
```

Comment:
Can you add `x ≤ sin x` for `x ≤ 0` too? Or is the claim that no one ever cares?

Reply:
Pretty sure no one cares but they are easy enough to add.